### PR TITLE
makefile: minor changes for code format checking

### PR DESCRIPTION
**Pull request by andelf**

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM
PTAL @BusyJay

File: Makefile [hunk 1]
```diff
@@ -69,12 +69,17 @@ test:
...
 
 bench:
-	LOG_LEVEL=ERROR RUST_BACKTRACE=1 cargo bench --features "${ENABLE_FEATURES}" -- --nocapture && \
+	LOG_LEVEL=ERROR RUST_BACKTRACE=1 cargo bench --features "${ENABLE_FEATURES}" -- --nocapture
 	RUST_BACKTRACE=1 cargo run --release --bin bench-tikv --features "${ENABLE_FEATURES}"
 
 format:
-	@cargo fmt -- --write-mode diff | grep -E "Diff .*at line" > /dev/null && cargo fmt -- --write-mode overwrite || exit 0
-	@rustfmt --write-mode diff tests/tests.rs benches/benches.rs | grep -E "Diff .*at line" > /dev/null && rustfmt --write-mode overwrite tests/tests.rs benches/benches.rs || exit 0
+	find . -iname '*.rs' | xargs cargo fmt -- --write-mode overwrite || true
```
Comment (BusyJay) [inline Makefile]: It will overwrite the file no matter if it needs to be modified.

Comment (andelf) [inline Makefile]: so there's a format-check

Comment (BusyJay) [inline Makefile]: Then format-check should not output any diff message. And `all` target should not depend on the format target.

Comment (andelf) [inline Makefile]: ok, close this

**Pull request closed without merging**